grpc_server.go

```diff
@@ -86,27 +75,43 @@ func (s *server) Get(ctx context.Context, in *hegel.GetRequest) (*hegel.GetRespo
 }
 
 func (s *server) Subscribe(in *hegel.SubscribeRequest, stream hegel.Hegel_SubscribeServer) error {
-	active_subscription.Inc()
-	defer active_subscription.Dec()
+	metrics.TotalSubscriptions.Inc()
+	metrics.Subscriptions.WithLabelValues("initializing").Inc()
+
+	logger := s.log.With("op", "subscribe")
+
+	initError := func(err error) error {
+		logger.Error(err)
+		metrics.Subscriptions.WithLabelValues("initializing").Dec()
+		labels := prometheus.Labels{
+			"op":    "subscribe",
+			"state": "initializing",
+		}
+		metrics.Errors.With(labels).Inc()
```

Comment:
why the `labels` var? Don't really need to change if you don't want though.

Reply:
ah, you're right. Would be nice to keep it consistent.